cert/selfsigned_test.go

```diff
@@ -26,7 +26,7 @@ func TestIsOutdatedCert(t *testing.T) {
 	// Generate TLS files with two extra IPs and domains.
 	err = cert.GenCertPair(
 		"lnd autogenerated cert", certPath, keyPath, extraIPs[:2],
-		extraDomains[:2], cert.DefaultAutogenValidity,
+		extraDomains[:2], false, cert.DefaultAutogenValidity,
```

Comment:
These changes should be included in the previous commit, otherwise it won't compile.

Reply:
Combined the commits for this reason.